.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         os:
           - 'centos-7'
           - 'centos-8'
-          - 'fedora-31'
+          - 'fedora-32'
```

Comment:
```suggestion
          - 'fedora'
```
Lets make this just match on fedora :+1:  then we won't have to worry in future?

Reply:
Must admit I'd not read the docs so didn't realise it could match on partials. Sounds a plan I'll stick that in.